pwnlib/scaffolds/remote.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+
+class RemoteExploitTemplate():
```

Comment:
I see no point in declaring new classes, especially without any base / interface type.

Also, I don't see anything provided here which is not handled better and in a more standardized fashion with Mako templates.